compiler/crates/relay-config/src/project_config.rs

```diff
@@ -56,6 +56,20 @@ pub struct RemotePersistConfig {
         deserialize_with = "deserialize_semaphore_permits"
     )]
     pub semaphore_permits: Option<usize>,
+
+    #[serde(default)]
+    pub authorization_header: Option<AuthorizationCommand>,
+}
```

Comment:
what do you think about adding new configuration section similar to `params`, but for `headers`:

And the dynamic values to these headers can be passed from the `relay.config.js` - dynamic JS config where you can call to that authorization command to generate tokens/headers.

Reply:
That's a much simpler solution - thanks for pointing it out. Updated to that